scripts/release.sh

```diff
@@ -27,11 +41,15 @@ rm -rf ${WORKDIR}/${DIR}/Vagrantfile
 rm -rf ${WORKDIR}/${DIR}/.idea
 rm -rf ${WORKDIR}/${DIR}/Makefile
 rm -rf ${WORKDIR}/${DIR}/.git*
+rm -rf ${WORKDIR}/${DIR}/.scrutinizer.yml
+rm -rf ${WORKDIR}/${DIR}/.checkignore.yml
 rm -rf ${WORKDIR}/${DIR}/scripts
 rm -rf ${WORKDIR}/${DIR}/pylintrc
 rm -rf ${WORKDIR}/${DIR}/extras
 rm -rf ${WORKDIR}/${DIR}/safe/test
 rm -rf ${WORKDIR}/${DIR}/realtime
+rm -rf ${WORKDIR}/${DIR}/bin
+rm -rf ${WORKDIR}/${DIR}/headless
```

Comment:
Hi @Gustry yes, we have already removed docs folder also, just open the whole file. And test_* is also deleted.